src/app.tsx

```diff
@@ -11,7 +11,7 @@ import { Routes } from './routes';
 import { asyncComponent } from './utils/asyncComponent';
 
 const ProvidersModal = asyncComponent(() =>
-  import(/* webpackChunkName: "providersModal" */ './pages/providersModal')
+  import(/* webpackChunkName: "providersModal" */ './features/onboardingWizard/components/wizard')
```

Comment:
I was trying a new idea that instead of scatter the code for a specific feature (in the store, components and pages) put it in one place and so it will be easy to remove or clean the code if necessary.

If you want me to move the pieces to `src/{store,components,pages}` let me know. It is not a big deal.

Regarding the providersModal, once the AWS part is done, I think it's a good idea to remove it.

Reply:
I've seen both approaches. For example, Cloud Metering structures the code (ducks-style) this way. Doug suggested that the structure we have now (rails-style?) follows best practices, but don't believe there is a single answer -- what works for one application may fall apart elsewhere. I don't feel strongly one way or the other, just that we are consistent.